modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro

```diff
@@ -112,12 +112,13 @@ definition {
 		if (${createRequest} == "true") {
 			Button.click(button = "Create a Request");
 		}
-		else {
+
+		if (${done} == "true") {
 			Button.click(button = "Done");
 		}
 	}
 
-	@summary = "Add a new organization"
+	@summary = "Go to create an organization"
```

Comment:
Hey, @juliosilveira77 could you please set a better summary?
reading full context this one sounds vague.
Suggestion: "Add new organization in organization page"